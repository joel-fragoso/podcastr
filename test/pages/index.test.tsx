import React from 'react'
import { render } from '../testUtils'
import HomeRoute from '../../src/pages/index'

describe('HomeRoute page', () => {
  it('should be able to matches snapshot', () => {
    const { asFragment } = render(<HomeRoute />, {})
    expect(asFragment()).toMatchSnapshot()
  })
})
