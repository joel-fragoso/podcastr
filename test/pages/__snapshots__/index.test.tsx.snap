// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRoute page should be able to matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="styles__LayoutRoot-sc-5fac4f-0 bFA-DUM"
  >
    <div
      class="styles__LayoutWrapper-sc-5fac4f-1 jHkEUP"
    >
      <div
        class="styles__LayoutContainer-sc-5fac4f-2 ejZZHs"
      >
        <div
          class="styles__LayoutContent-sc-5fac4f-3 gBLEzg"
        >
          <main
            class="styles__Container-sc-1gl7fhg-0 dVxcFv"
          >
            <h1
              class="styles__HelloWorld-sc-1gl7fhg-1 cagzVN"
            >
              Olá mundo 🎉️!
            </h1>
          </main>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
